.github/workflows/deploy-docs.yaml

```
@@ -1,14 +1,19 @@
 name: Auto-deployment of bertron-schema Documentation
 on:
+  #pull_request:
+  #  branches:
+  #    - main
```

I suggest uncommenting these lines and reformatting them to match the `push` block below.

```suggestion
  pull_request:
    branches: [ main ]
```

I don't know whether uncommenting this will cause the workflow to run in this PR (since the block does not exist on the `main` branch yet), but uncommenting it _before merging this PR branch in_ will put it on the `main` branch so it can take effect for (at least) future PRs.